app/app.go

```diff
@@ -974,6 +986,9 @@ func (app *App) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci.R
 	resp.ConsensusMessagesResponses = consensusResponses
 
 	drsVersion, err := dymintversion.GetDRSVersion()
+	if app.dymintVersionGetter != nil {
+		drsVersion, err = app.dymintVersionGetter()
+	}
```

Comment:
why the fallback here? in which cause is it being called?

Reply:
it's for mocking when testing